backstopper-jaxrs/build.gradle

```diff
@@ -0,0 +1,19 @@
+evaluationDependsOn(':')
+
+dependencies {
+    compile(
+            project(":backstopper-servlet-api"),
+            project(":backstopper-jackson"),
+            "javax.ws.rs:javax.ws.rs-api:2.0.1",
```

Comment:
Got it. We're still a Maven shop, so wasn't 100% on the right way to do things with Gradle.